controllers/platformnetwork_controller.go

```diff
@@ -407,19 +471,31 @@ func (r *PlatformNetworkReconciler) FindExistingAddressPool(client *gophercloud.
 // the network resource since at the system level these are two independent
 // resources.
 func (r *PlatformNetworkReconciler) ReconcileAddressPool(client *gophercloud.ServiceClient, instance *starlingxv1.PlatformNetwork) error {
-	pool, err := r.FindExistingAddressPool(client, instance)
+	var pool *addresspools.AddressPool
+	var err error
+	pool, err = r.FindExistingAddressPool(client, instance)
```

Comment:
you can declare and assign the variables using `:=` (check the line 410 removed)